packages/SwingSet/src/kernel/state/kernelKeeper.js

```diff
@@ -317,6 +317,16 @@ export default function makeKernelKeeper(
     return ri;
   }
 
+  function setDefaultReapInterval(interval) {
+    assert(
```

Comment:
I was thinking this might be tricky, but I just looked at the `nat` package and discovered it also has an `isNat` predicate, which will be the thing to use here since we also need to allow `'never'`.